pages/profile-setup/index.page.tsx

```diff
@@ -1,3 +1,56 @@
-export default function ProfileSetup() {
-  return <div>프로필 입력 페이지입니다.</div>;
+import { GetServerSidePropsContext } from "next";
+
+import Image from "next/image";
+
+import classNames from "classnames/bind";
+
+import ProfileSetupForm from "./components/ProfileSetupForm/ProfileSetupForm";
+import { instance } from "@/apis/axios";
+
+import styles from "./index.module.scss";
+
+const cn = classNames.bind(styles);
+
+interface ProfileSetupProps {
+  accountId: number;
+  email: string;
+}
+
+export const getServerSideProps = async (context: GetServerSidePropsContext) => {
+  const accountId = context.query.id;
+
+  const res = await instance.get(`/account-info/${accountId}`);
+  const { email } = res.data;
+
+  return {
+    props: { accountId, email },
+  };
+};
+
+export default function ProfileSetup({ accountId, email }: ProfileSetupProps) {
+  return (
+    <>
+      <header className={cn("header")}>
+        <Image src="/icons/tickita-logo.svg" alt="logo" width={60} height={22} />
+      </header>
+      <main>
+        <div className={cn("container")}>
+          <div className={cn("profile-container")}>
+            <p className={cn("description")}>
+              티키타에 합류하신 걸 환영해요
+              <br />
+              원활한 사용을 위해 추가 정보를 입력해주세요
+            </p>
+            <Image
+              src="/icons/profile-add-icon.svg"
+              alt="default profile image"
+              width={120}
+              height={120}
+            />
```

Comment:
```suggestion
            <input id="profileImage" type="file" accept="이미지 확장자들"/>
            <lable htmlFor="profileImage">
               <Image src="{imageUrl ? imageUrl : "기본 두더지 이미지"}" alt="유저 프로필 이미지" width={120} height={120}>
            </label>
```
p3: 지금 유저 이미지 넣는 부분이 없는 것 같아요 input, label 추가해서 이미지 넣는 로직도 있어야 할 것 같네요!
기본 두더지 이미지랑 카메랑 이미지를 한번에 넣으셨는데 분리하셔서 유저가 프로필 이미지 추가했을 때도 고려하시면 좋을 것 같아요

Reply:
`아직 프로필 이미지 추가 기능은 없습니다. 바로 이어서 구현 진행 예정입니다.` 
이 내용을 리뷰하고 봤네요... 참고만 해주시면 될 것 같아요~!